camcan/utils/bagging.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from joblib import delayed, Parallel
+from sklearn.base import BaseEstimator, RegressorMixin
+from sklearn.linear_model import RidgeCV
+from sklearn.model_selection._split import check_cv, PredefinedSplit
```

Comment:
from sklearn.model_selection import check_cv, PredefinedSplit

should be enough

Reply:
ok